test/migrator/connection_test.rb

```diff
@@ -0,0 +1,115 @@
+require 'test_helper'
```

Comment:
Huumm all tests outside this block https://github.com/lotus/model/pull/220/files#diff-470f4a122e61995dd74573944a76467aR46 cover non-jdbc connections :)